<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ApplicationSettingsWidgetSecurity</class>
 <widget class="QWidget" name="ApplicationSettingsWidgetSecurity">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>595</width>
    <height>541</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Timeouts</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QCheckBox" name="clearClipboardCheckBox">
        <property name="text">
         <string>Clear clipboard after</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="clearClipboardSpinBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="suffix">
         <string comment="Seconds"> sec</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>999</number>
        </property>
        <property name="value">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="clearSearchCheckBox">
        <property name="text">
         <string>Clear search query after</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="clearSearchSpinBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="suffix">
         <string comment="Minutes"> min</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>1440</number>
        </property>
        <property name="value">
         <number>5</number>
        </property>
        <property name="displayIntegerBase">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="lockDatabaseIdleCheckBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Lock databases after inactivity of</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="lockDatabaseIdleSpinBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="suffix">
         <string comment="Seconds"> sec</string>
        </property>
        <property name="minimum">
         <number>10</number>
        </property>
        <property name="maximum">
         <number>43200</number>
        </property>
        <property name="value">
         <number>240</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="touchIDResetCheckBox">
        <property name="text">
         <string>Forget TouchID after inactivity of</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="touchIDResetSpinBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="suffix">
         <string> min</string>
        </property>
        <property name="maximum">
         <number>1440</number>
        </property>
        <property name="value">
         <number>30</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Convenience</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="lockDatabaseOnScreenLockCheckBox">
        <property name="text">
         <string>Lock databases when session is locked or lid is closed</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="touchIDResetOnScreenLockCheckBox">
        <property name="text">
         <string>Forget TouchID when session is locked or lid is closed</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="lockDatabaseMinimizeCheckBox">
        <property name="text">
         <string>Lock databases after minimizing the window</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="relockDatabaseAutoTypeCheckBox">
        <property name="text">
         <string>Re-lock previously locked database after performing Auto-Type</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="passwordRepeatCheckBox">
        <property name="text">
         <string>Don't require password repeat when it is visible</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="passwordCleartextCheckBox">
        <property name="text">
         <string>Don't hide passwords when editing them</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="passwordShowDotsCheckBox">
        <property name="text">
         <string>Don't use placeholder for empty password fields</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="passwordPreviewCleartextCheckBox">
        <property name="text">
         <string>Hide passwords in the entry preview panel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="hideNotesCheckBox">
        <property name="text">
         <string>Hide entry notes by default</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="privacy">
     <property name="title">
      <string>Privacy</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QCheckBox" name="fallbackToSearch">
        <property name="text">
         <string>Use DuckDuckGo service to download website icons</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>30</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
